external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -3708,23 +3677,27 @@ pub fn on_go_to_def_request(context: &Context, request: &Request, symbols: &Symb
         col,
         request.id.clone(),
         |u| {
-            // TODO: Do we need beginning and end of the definition? Does not seem to make a
-            // difference from the IDE perspective as the cursor goes to the beginning anyway (at
-            // least in VSCode).
-            let range = Range {
-                start: u.def_loc.start,
-                end: u.def_loc.start,
-            };
-            let path = symbols.file_name_mapping.get(&u.def_loc.fhash).unwrap();
-            let loc = Location {
-                uri: Url::from_file_path(path).unwrap(),
-                range,
-            };
+            let loc = def_ide_location(&u.def_loc, symbols);
             Some(serde_json::to_value(loc).unwrap())
         },
     );
 }
 
+pub fn def_ide_location(def_loc: &DefLoc, symbols: &Symbols) -> Location {
```

Comment:
Not a strictly necessary one but I hope we will be able to ultimately support locations in inlay type hints for additional information to be displayed (that's why this was introduced in the first place).